index.html

```diff
@@ -411,6 +411,14 @@ <h4>Correctness properties</h4>
 </div>
 
 
+<h4>Detecting timelock mixing</h4>
+<p>
+The <samp>nSequence</samp> field in a transaction input, or the <samp>nLockTime</samp> field in transaction can be specified either as a time or height but not both.
+Therefore it is not possible to spend scripts that require satisfaction of both, height based timelock and time based timelock of the same type.
+<ul>
+<li><b>"k"</b> No timelock mixing. This expression does not contain a mix of heightlock and timelock of the same type. If the miniscript does not have the "k" property, the miniscript template will not match the user expectation of the corresponding spending policy.</li>
```

Comment:
Mention the other types here too, or alternatively, drop them from `Props()` below?

Reply:
Dropped from `Props()` as other properties are just implementation detail to obtain `k`

Reply:
nit: shouldn't this be part of the above list with the other properties? (`"z"`, `"o"`, `"n"`, `"d"`, `"u"`)